core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -73,44 +74,46 @@ class AnalyzerManager(
       withExistingModuleFor(fileInfo, req)((req, moduleId) =>
         getOrSpawnNew(moduleId) forward req)
     case req @ TypecheckFilesReq(files) =>
-      if (files.exists(config.findProject(_).isEmpty))
-        sender ! EnsimeServerError("Update .ensime file.")
-      else {
-        val original = sender
-        val filesPerProject = files.groupBy(config.findProject(_)).map(x => x._1.get -> x._2)
+      val original = sender
+      val filesPerProject = files.groupBy(config.findProject(_)).map(x => x._1 -> x._2)
```

Comment:
I know this was in the original code, but `._` getters are discourage. Prefer ` { case (a, b) => } ` and give meaningful names to `a` and `b`.

Ummm, is `map(x => x._1 -> x._2)` even doing anything? Can't we just drop the `map`?